app/models/item.rb

```diff
@@ -0,0 +1,5 @@
+class Item < ApplicationRecord
+  has_many :invoice_items, dependent: :destroy
```

Comment:
The order might have to be switched?

Reply:
@suzkiee When Netia and I were talking through this we discovered that InvoiceItems looked really similar to a Join table, so that's why this was added in Item and Invoice models, so Items could know about its Invoices and vice versa